src/control/Control.cpp

```diff
@@ -3028,8 +3028,8 @@ void Control::clipboardPasteImage(GdkPixbuf* img)
 	auto image = new Image();
 	image->setImage(img);
 
-	int width = gdk_pixbuf_get_width(img);
-	int height = gdk_pixbuf_get_height(img);
+	auto width = static_cast<double>(gdk_pixbuf_get_width(img)) / settings->getDisplayDpi() * 72.0;
```

Comment:
Why `72.0`? This should've been documented in a comment, or the constant should've been placed into an appropriately named `const` variable.

Reply:
I've looked into that, it is used everywhere in our codebase and brings the display dpi into relation to our pageView. Basically we assume a dpi of 72 as 100%.

Reply:
It is better to change this in a slip, but separated from this fix. We can write this out as a simple issue, a beginner can fix.